x-pack/plugins/enterprise_search/public/applications/app_search/components/api_logs/api_logs.tsx

```diff
@@ -75,6 +76,7 @@ export const ApiLogs: React.FC<Props> = ({ engineBreadcrumb }) => {
           <EuiSpacer size="m" />
 
           <ApiLogsTable hasPagination />
+          <ApiLogFlyout />
```

Comment:
I debated putting `ApiLogFlyout` within `ApiLogsTable` but opted not to in the end. I don't feel _super_ strongly either way, but I landed on this side of the fence because:

- I like our `<Table />` components just being self contained modular tables/components, and it definitely simplifies tests for the table component
- It's nice having an 'overview' of what's on the page from the view itself

That being said in the end I don't think it super matters either way so happy to be convinced otherwise